src/ipfs-npm/registry/clone/index.js

```diff
@@ -1,30 +1,52 @@
 'use strict'
 
-const config = require('../config')
-const log = config.log
 require('dnscache')({ enable: true })
 const follow = require('follow-registry')
 const patch = require('patch-package-json')
 const fs = require('graceful-fs')
 const timethat = require('timethat').calc
-const http = require('http-https')
-const url = require('url')
+const request = require('request')
 const IBS = require('ipfs-blob-store')
+const multiaddr = require('multiaddr')
+const series = require('async/series')
+
 const ModuleWriter = require('./module-writer')
 const Verifier = require('./verifier')
+const config = require('../../config')
+const log = config.log
 
-var latestSeq = 'unknown'
+let latestSeq = 'unknown'
 // const GLOBAL_INDEX = '-/index.json'
 // const NOT_FOUND = '-/404.json'
 
 module.exports = RegistryClone
 
-function RegistryClone (ipfs, seqNumber) {
+function RegistryClone (ipfs, opts) {
   if (!(this instanceof RegistryClone)) {
-    return new RegistryClone(ipfs, seqNumber)
+    return new RegistryClone(ipfs, opts)
+  }
+
+  let bsConfig = config.blobStore
+  opts = opts || {}
+
+  if (opts.url) {
+    const parsed = multiaddr(opts.url).nodeAddress()
+    bsConfig = Object.assign(bsConfig, {
+      host: parsed.address,
+      port: parsed.port
+    })
+  }
+
+  if (typeof opts.flush === 'boolean') {
+    bsConfig.flush = opts.flush
+  }
+
+  if (opts.seqNumber) {
+    // TODO: what to do?
```

Comment:
start from that seq number forward.

seqnumber is a stored value on disk, so that it is easy to resume